### PR TITLE
[ES] Version Control

### PR DESCRIPTION
**Pull request by MartinEliasQ**
### Describe your changes

- Added the translated version of version control

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/version-control.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## ¿Qué es?
+
+El control de versiones (o control de fuente) es la práctica de rastrear y gestionar los cambios en un documento.
+Es un sistema que registra cambios en un archivo o conjunto de archivos a lo largo del tiempo, de manera que puedas recuperar versiones específicas más adelante.
+
+## Problema que aborda
+
+Los sistemas de control de versiones funcionan para abordar los siguientes problemas:
+respaldar un documento o base de código a medida que cambia con el tiempo,
+permitir a múltiples usuarios resolver conflictos cuando hay cambios superpuestos, y
```
Comment (ramrodo) [inline content/es/version-control.md:18]:
```suggestion
permitir a múltiples usuarios resolver conflictos cuando hay cambios superpuestos y
```

File: content/es/version-control.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+almacenar un registro de los cambios a lo largo del tiempo.
+Dado que el código de la aplicación a menudo es complejo y vital para los procedimientos de negocio esenciales, resulta relevante seguir la pista de quién hizo qué cambio, cuándo se produjo y la razón del mismo.
+Además, muchas, si no la mayoría de las aplicaciones, son modificadas por varios desarrolladores, y a menudo existen conflictos entre los cambios introducidos por diferentes desarrolladores.
```
Comment (ramrodo) [inline content/es/version-control.md:21]:
```suggestion
Además, muchas, si no la mayoría de las aplicaciones, son modificadas por varios desarrolladores y a menudo existen conflictos entre los cambios introducidos por diferentes desarrolladores.
```

File: content/es/version-control.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## ¿Cómo ayuda?
+
+El control de versiones ayuda a los desarrolladores a avanzar rápidamente y a mantener la eficiencia, al mismo tiempo que guarda un registro de los cambios y ofrece una herramienta para resolver conflictos.
+Esta herramienta facilita la tarea de guardar el código de una aplicación en un repositorio, lo que simplifica el trabajo en equipo. Hoy en día, para desarrollar aplicaciones modernas, es muy común depender de sistemas de control de versiones, como git, que se utilizan para almacenar este código.
```
Comment (ramrodo) [inline content/es/version-control.md:26]:
```suggestion
Esta herramienta facilita la tarea de guardar el código de una aplicación en un repositorio, lo que simplifica el trabajo en equipo. Hoy en día, para desarrollar aplicaciones modernas, es muy común depender de sistemas de control de versiones, como git, que se utiliza para almacenar este código.
```

**Review by ramrodo — COMMENTED** [review]
¡Gracias! Solo unos pequeños comentarios.
(disculpa la tardanza en la revisión)

**Comment by MartinEliasQ** [comment]
> ¡Gracias! Solo unos pequeños comentarios @MartinEliasQ (disculpa la tardanza en la revisión)

Gracias a ti @ramrodo !

**Review by ramrodo — APPROVED** [review]
LGTM

**Review by electrocucaracha — APPROVED** [review]
/lgtm

**Pull request merged**